src/static/js/change_user_settings.js

```diff
@@ -190,6 +199,13 @@ $(function() {
                 $("#quicklinksid").find("option[value='" + common.trim(v + "']")).attr("selected", "selected");
             });
             $("#quicklinksid").change();
+
+            let fr = config.str(asm.user + "_FavouriteReportsID").split(",");
+            $.each(fr, function(i, v) {
+                $("#favouritereportsid").find("option[value='" + common.trim(v + "']")).attr("selected", "selected");
```

Comment:
isn't v here in ID=Title format? So you'll need to throw away the =Title part when finding the value to select

Reply:
Already sorted.